src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1334,7 +1336,7 @@ public function getParameterBag()
 
 EOF;
         if (!$this->asFiles) {
-            $code = preg_replace('/^.*buildParameters.*\n.*\n.*\n/m', '', $code);
+            $code = preg_replace('/^.*buildParameters.*\n.*\n.*\n\n?/m', '', $code);
```

Comment:
Should be reverted as it does not look related to the purpose of this PR, is it?

Reply:
This class generates implementations of the `ContainerInterface` that has been changed with this PR. This is why I've adjusted the code template above to add the new type declarations and remove the now-obsolete type cast.

This `preg_replace` call removes an `if` block from the generated code under certain circumstances. Because I've removed the type cast, this operation would leave an empty line at the beginning of the method, like this:

```php
public function hasParameter(string $name)
{

    return isset($this->parameters[$name]) || isset($this->loadedDynamicParameters[$name]) || array_key_exists($name, $this->parameters);
}
```

I've adjusted the regular expression, so that this extra new line is also removed in case its there.